src/proxy/processors/push-action/parsePush.ts

```diff
@@ -99,9 +99,9 @@ const getCommitData = (contents: CommitContent[]) => {
       const indexOfMessages = formattedContent.indexOf('');
       console.log({ indexOfMessages });
 
-      const message = formattedContent
-        .slice(indexOfMessages + 1, formattedContent.length - 1)
-        .join(' ');
+      const message = formattedContent 
+        .slice(indexOfMessages + 1) 
+        .join(' ').trim(); 
```

Comment:
This is the main change in this PR - the rest is all related to testing - although there is no direct testing of pushParsing currently implemented. 

The change seems to resolve the issue for me on the PR where we had this issue (which was contributed without the problem message already).

Reply:
Added tests for push parsing - can you have another look?